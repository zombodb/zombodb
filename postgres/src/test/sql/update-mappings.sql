SELECT zdb_update_mapping('so_posts');

/* now make sure the things that are important haven't changed */
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_meta'->>'primary_key' = 'id';
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_all'->>'analyzer' = 'phrase';
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_source'->>'enabled' = 'false';
