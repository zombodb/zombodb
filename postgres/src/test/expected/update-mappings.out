SELECT zdb_update_mapping('so_posts');
 zdb_update_mapping 
--------------------
 
(1 row)

/* now make sure the things that are important haven't changed */
SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_meta'->>'primary_key' = 'id';
 ?column? 
----------
 t
(1 row)

SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_all'->>'analyzer' = 'phrase';
 ?column? 
----------
 t
(1 row)

SELECT zdb_get_index_mapping('so_posts')->'mappings'->'data'->'_source'->>'enabled' = 'false';
 ?column? 
----------
 t
(1 row)

